### PR TITLE
Cumulus 1775

### PR DESCRIPTION
**Pull request by Jkovarik**
**Summary:** Summary of changes

Addresses [CUMULUS-1775: Develop amazing new feature](https://bugs.earthdata.nasa.gov/browse/CUMULUS-1775)

## Changes

* Detailed list or prose of changes
* ...

## PR Checklist

- [x] Update CHANGELOG
- [N/A ] Unit tests
- [x] Adhoc testing
- [ N/A ] Integration tests

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -243,6 +243,9 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 - **CUMULUS-1768**
   - Fix an issue in the stats endpoint in `@cumulus/api` to send back stats for the correct type
 
+- **CUMULUS-1775**
```
Comment (laurenfrederick) [inline CHANGELOG.md]: This looks like it's under 1.19. Can you please move to the unreleased section?

Comment (Jkovarik) [reply CHANGELOG.md]: Oops, bad merge presumably.   I'll sort it.

Comment (Jkovarik) [reply CHANGELOG.md]: Updated: dcfff5b53

File: CHANGELOG.md [hunk 2]
```diff
@@ -6,10 +6,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
-### BREAKING CHNAGES
+### BREAKING CHANGE
 
 - **CUMULUS-1714**
-  - Changed the format of the message sent to the granule SNS Topic. Message includes the granule record under `record` and the type of event under `event`. Messages with `deleted` events will have the record that was deleted with a `deletedAt` timestamp. Options for `event` are `Create | Update | Delete`
+  - Changed the format of the message sent to the granule SNS Topic. Message includes the granule record under `record` and the type of event under `event`. Messages with `deleted` events will have the record that was deleted with a `deletedAt` timestamp. Options for `event` are `Create | UpdatSe | Delete`
```
Comment (laurenfrederick) [inline CHANGELOG.md]:
```suggestion
  - Changed the format of the message sent to the granule SNS Topic. Message includes the granule record under `record` and the type of event under `event`. Messages with `deleted` events will have the record that was deleted with a `deletedAt` timestamp. Options for `event` are `Create | Update | Delete`
```

File: CHANGELOG.md [hunk 3]
```diff
@@ -6,10 +6,15 @@ The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/).
 
 ## [Unreleased]
 
-### BREAKING CHNAGES
+### BREAKING CHANGE
```
Comment (laurenfrederick) [inline CHANGELOG.md]:
```suggestion
### BREAKING CHANGES
```

**Pull request merged**